backend/uclapi/timetable/management/commands/update_gencache.py

```diff
@@ -342,4 +343,13 @@ def handle(self, *args, **options):
         # to be run again in the future.
         self._redis.delete(cache_running_key)
 
+        url = "https://cachet.apps.uclapi.com/api/v1/components/2"
+        cachet_token = settings.CACHET_KEY
+        cachet_format = "application/x-www-form-urlencoded"
+        payload_headers = {"X-Cachet-Token": cachet_token,
+                           "Content-Type": cachet_format}
+        payload = {"status": 1}
+        requests.request("PUT", url, data=payload, headers=payload_headers)
```

Comment:
It looks like we can just use `requests.put` which might be a bit neater?

Reply:
Yep let me stick that in